edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/MetricsModule.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Service
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Autofac;
+    using Microsoft.Azure.Devices.Edge.Agent.Core.Metrics;
+    using Microsoft.Azure.Devices.Edge.Util.Metrics;
+    using Microsoft.Azure.Devices.Edge.Util.Metrics.NullMetrics;
+    using Microsoft.Azure.Devices.Edge.Util.Metrics.Prometheus.Net;
+
+    public class MetricsModule : Autofac.Module
```

Comment:
why Autofac here? "Module" name conflict?

Reply:
Yes, although removing it now doesn't conflict. Removed